Dockerfile.template

```diff
@@ -19,15 +40,15 @@ RUN if [ "%%RESIN_MACHINE_NAME%%" = "raspberrypi3" ]; then export EXTRA_PACKAGES
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Get the edge-node-manager binary, rename and make executable
-RUN TAG=$(curl https://api.github.com/repos/resin-io/edge-node-manager/releases/latest -s | jq .tag_name -r) && \
-    echo "Pulling $TAG of the edge-node-manager binary for %%RESIN_ARCH%%" && \
-    curl -k -O https://resin-production-downloads.s3.amazonaws.com/edge-node-manager/$TAG/edge-node-manager-$TAG-linux-%%RESIN_ARCH%% && \
-    mv edge-node-manager-$TAG-linux-%%RESIN_ARCH%% edge-node-manager && \
-    chmod +x edge-node-manager
+# RUN TAG=$(curl https://api.github.com/repos/resin-io/edge-node-manager/releases/latest -s | jq .tag_name -r) && \
+#     echo "Pulling $TAG of the edge-node-manager binary for %%RESIN_ARCH%%" && \
+#     curl -k -O https://resin-production-downloads.s3.amazonaws.com/edge-node-manager/$TAG/edge-node-manager-$TAG-linux-%%RESIN_ARCH%% && \
+#     mv edge-node-manager-$TAG-linux-%%RESIN_ARCH%% edge-node-manager && \
+#     chmod +x edge-node-manager
```

Comment:
We should remove this rather than comment it out.

Reply:
Done :)